samples/Mvc.Client/Startup.cs

```diff
@@ -61,6 +61,17 @@ public void ConfigureServices(IServiceCollection services)
                 options.Scope.Add("user:email");
             })
 
+            .AddWorkWeixin(options =>
+            {
+                options.ClientId = Configuration["WorkWeixin:ClientId"];
+                options.AgentId = Configuration["WorkWeixin:AgentId"];
+                options.ClientSecret = Configuration["WorkWeixin:ClientSecret"];
+            })
+            .AddWeixin(options =>
+            {
+                options.ClientId = Configuration["Weixin:ClientId"];
+                options.ClientSecret = Configuration["Weixin:ClientSecret"];
+            })
```

Comment:
Please revert - we can't continuously add new providers to the sample, it will become too unwieldy.

```suggestion
```